action_server/frontend/src/components/Code.tsx

```diff
@@ -60,15 +62,16 @@ export const Code: FC<CodeProps> = ({
   }, [autoFocus]);
 
   return (
-    <Box height={height} mb="$24">
+    <Box key="base-code-input-wrapper" height={height} mb="$24">
```

Comment:
No need/effect for a key in a single item component
```suggestion
    <Box height={height} mb="$24">
```